R/drive_upload.R

```diff
@@ -179,17 +176,17 @@ build_drive_upload <- function(input = NULL,
     id <- proc_res$id
   }
 
-  url <- file.path(.drive$base_url,
-                   "upload/drive/v3/files",
-                   paste0(id, "?uploadType=media"))
-
-  list(
-    method = "PATCH",
-    url = url,
+  build_request(
+    path = "/upload/drive/v3/files/{fileId}",
```

Comment:
If I'm following you (I've also read the comment below), YES we should create distinct endpoints. So that identification by endpoint tells you exactly what `path` should be, exactly what the http verb is, what params are allowed, etc.